pkg/liqoctl/install/utils.go

```diff
@@ -89,9 +89,21 @@ func installOrUpdate(ctx context.Context, helmClient *helm.HelmClient, k provide
 			Wait:             true,
 		}
 
-		_, err = helmClient.InstallOrUpgradeChart(ctx, &chartSpec)
-		if err != nil {
-			return err
+		// provide the possibility to exit installation on context cancellation
+		errCh := make(chan error)
+		defer close(errCh)
+		go func() {
+			_, err = helmClient.InstallOrUpgradeChart(ctx, &chartSpec)
```

Comment:
Hoped the helmClient automatically terminated when the context is canceled, to simplify the entire logic.

Reply:
Unfortunately not, it doesn't propagate the context in the install function. And the helm library does the same thing